apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -153,7 +154,9 @@ class ScriptEditor extends React.Component {
       lessonDescriptions: this.props.i18nData.stageDescriptions,
       teacherResources: resources,
       hasImportedLessonDescriptions: false,
-      oldScriptText: this.props.initialLessonLevelData
+      oldScriptText: this.props.initialLessonLevelData,
+      includeStudentFacingLessonPlans:
```

Comment:
I think in order to make all the places we are talking about this shorter I think we can probably call them "student lesson plans" and leave out the facing part. This comment probably applies to all the files